$NetBSD: patch-af,v 1.4 2001/04/08 16:25:28 dmcmahill Exp $

libtool needs a tag for f77/f2c
also be sure to link the fortran libs

--- Makemex.in.orig	Tue May  9 06:46:04 2000
+++ Makemex.in	Sun Apr  1 21:44:35 2001
@@ -20,4 +20,5 @@
 LDFLAGS = @LDFLAGS@
 LIBS = @LIBS@
+FLIBS = @FLIBS@
 
 INCLUDES= -I$(SCIDIR)/routines 
@@ -30,7 +31,7 @@
 
 LTCOMPILE = $(LIBTOOL) --mode=compile $(QUIET) $(CC) $(INCLUDES) $(CPPFLAGS) $(CFLAGS)
-FLTCOMPILE = $(LIBTOOL) --mode=compile $(QUIET) $(FC) $(FFLAGS)
+FLTCOMPILE = $(LIBTOOL) --tag=CC --mode=compile $(QUIET) $(FC) $(FFLAGS)
 
-LINK = $(LIBTOOL) --mode=link $(QUIET)  $(CC) $(CFLAGS) $(LDFLAGS) -o $@
+LINK = $(LIBTOOL) --mode=link $(QUIET)  $(CC) $(CFLAGS) $(LDFLAGS) $(FLIBS) -o $@
 
 OBJECTS = $(LIBRARY)_gateway.lo $(OTHEROBJECTS:.o=.lo)
